.hooks/pre-commit

```diff
@@ -4,4 +4,17 @@ set -e
 
 # consider using pre-push instead
 
+# get all staged files
+staged_files=$(git diff --cached --name-only)
+
+# if en_US.json is among staged files, sync i18n
+if echo "$staged_files" | grep -q "lang/en_US.json"; then
+    printf "\n🌎 Changes detected in the en_US locale."
+    printf "\n⏳ pnpm i18n:sync\n"
+    pnpm i18n:sync
```

Comment:
Trying to run this command generates a permission error in my VM. It looks like sail owns the `livewire-tmp` directory:
```
jamiras@ubuntu24 pr/2832:~/source/RAWeb$ pnpm i18n:sync

> raweb@ i18n:sync /home/jamiras/source/RAWeb
> sync-i18n --files '**/lang/*.json' --primary en_US --languages es_ES pl_PL pt_BR --space 4

/home/jamiras/source/RAWeb/node_modules/.pnpm/glob@7.2.3/node_modules/glob/sync.js:341
        throw er
        ^

Error: EACCES: permission denied, scandir '/home/jamiras/source/RAWeb/storage/app/livewire-tmp/lang'
...
```

Reply:
Interesting - I've pushed a change to the command so it's more specific about the targeted path. Can you try again with latest and see if the error is still thrown?

Reply:
That works.